src/components/ContentNode/Caption.vue

```diff
@@ -53,4 +58,8 @@ export default {
 /deep/ p {
   display: inline-block;
 }
+
+caption {
```

Comment:
I guess this is less of a concern, but if `caption-side: bottom;` is applied, this styling will not work as expected. However I am not sure how to check that with pure CSS. MDN jus suggest using padding around the caption...